src/module/add_todo.js

```diff
@@ -0,0 +1,17 @@
+import todos from './todo';
+
+const addTodos = (e) => {
+  const addTodo = document.querySelector('.add_todo');
+  const todoObject = {
+    description: addTodo.value,
+    completed: 'checked',
```

Comment:
- [ ] Please make sure the new task should have a property `completed` to set `false`. It is part of this project requirement: "By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5)."
- [ ] Please  pay attention to this screenshort
![1](https://user-images.githubusercontent.com/63972670/173175940-afc4cd5b-1301-4602-a6a2-7632d68e9e01.PNG)